### DISCUSSION TITLE
Support for OpenID SSO?

### DISCUSSION THREAD

**Discussion by e-renna** [tobychui/zoraxy#74] [Q&A]
Is SSO currently supported, or are there any plans to support it in the future?

**Comment by tobychui** [comment]
> Is SSO currently supported, or are there any plans to support it in the future?

@e-renna No, this is a reverse proxy not a centralize authentication gateway. However, if you want it, you can feel free to contribute some code into the project to make it happens, and I will be happy to test them out for you :D
